pkg/api/types.go

```diff
@@ -153,6 +153,15 @@ const (
 	TerminationMessagePathDefault string = "/dev/termination-log"
 )
 
+// There is no namespace required for these kinds specified here.
+// If a kind is not enumerated here, it is assumed to have a namespace scope.
+var NoneNamespacedKinds = []string{
```

Comment:
There needs to be a single source of truth for this, but I'm not sold on making a list here. Also it's not binary, some resources (events) are both.